CHANGELOG.md

```diff
@@ -5,22 +5,41 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 since 0.4.1, and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [0.7.0] -  2024-06-20
 
-### Fixed
-
-- Fix a deprecation warning in PySide6, thanks to [@naglis](https://github.com/naglis) ([issue #595](https://github.com/freedomofpress/dangerzone/issues/595))
-- Make update notifications work in systems with PySide2, thanks to [@naglis](https://github.com/naglis) ([issue #788](https://github.com/freedomofpress/dangerzone/issues/788))
-
-### Security
+### Added
 
 - Integrate Dangerzone with gVisor, a memory-safe application kernel, thanks to [@EtiennePerot](https://github.com/EtiennePerot) ([#126](https://github.com/freedomofpress/dangerzone/issues/126))
-  As a result of this integration, we have also improved Dangerzone's security
-  in the following ways:
+  As a result of this integration, we have also improved Dangerzone's security in the following ways:
   * Prevent attacker from becoming root within the container ([#224](https://github.com/freedomofpress/dangerzone/issues/224))
   * Use a restricted seccomp profile ([#225](https://github.com/freedomofpress/dangerzone/issues/225))
   * Make use of user namespaces ([#228](https://github.com/freedomofpress/dangerzone/issues/228))
+- Added some articles about the Dangerzone project to the `README.md` ([#807](https://github.com/freedomofpress/dangerzone/pull/807))
```

Comment:
In past releases, we've been trying to include changes that are of note either to our end users, or our contributors. Think of it as a way of staying up-to-date with the software. Some changes are quite small, and we don't tend to include them here, mostly to improve the signal to noise ratio.

So, I think based on the above we can omit this entry.

Reply:
Makes sense. Removed.